SPECS/buildah/buildah.spec

```diff
@@ -0,0 +1,1973 @@
+%global with_bundled 1
+
+
+%global with_debug 1
+
+
+%if 0%{?with_debug}
+%global _find_debuginfo_dwz_opts %{nil}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package   %{nil}
+%endif
+
+%global provider github
+%global provider_tld com
+%global project containers
+%global repo buildah
+# https://github.com/containers/buildah
+%global import_path %{provider}.%{provider_tld}/%{project}/%{repo}
+%global git0 https://%{import_path}
+
+# Used for comparing with latest upstream tag
+# to decide whether to autobuild (non-rawhide only)
+%define built_tag v1.18.0
+%define built_tag_strip %(b=%{built_tag}; echo ${b:1})
+%define download_url https://%{import_path}/archive/%{built_tag}.tar.gz
+
+Name:         %{repo}
+Version:      1.18.0
+Release:      6%{?dist}
+Summary:      A command line tool used for creating OCI Images
+License:      Apache-2.0
+Vendor:       Microsoft Corporation
+Distribution: Mariner
+URL: https://%{name}.io
+Source: %{download_url}#/%{name}-%{version}.tar.gz
+BuildRequires: device-mapper-devel
+BuildRequires: golang
+BuildRequires: git
+BuildRequires: glib2-devel
+BuildRequires: glibc-static >= 2.35-3%{?dist}
+BuildRequires: go-md2man
+BuildRequires: go-rpm-macros
+BuildRequires: gpgme-devel
+BuildRequires: libassuan-devel
+BuildRequires: make
+Requires: libcontainers-common
+BuildRequires: ostree-devel
+BuildRequires: btrfs-progs-devel
+
+
+#Requires: crun >= 0.10-1
+BuildRequires: libseccomp-static
+Recommends: container-selinux
+Requires: libseccomp >= 2.4.1-0
+Recommends: slirp4netns >= 0.3-0
+Recommends: fuse-overlayfs
+
+
+%description
+The %{name} package provides a command line tool which can be used to
+* create a working container from scratch
+or
+* create a working container from an image as a starting point
+* mount/umount a working container's root file system for manipulation
+* save container's root file system layer to create a new image
+* delete a working container or an image
+
+%prep
+%autosetup -Sgit -n %{name}-%{built_tag_strip}
+sed -i 's/GOMD2MAN =/GOMD2MAN ?=/' docs/Makefile
+sed -i '/docs install/d' Makefile
+
+%build
+mkdir _build
+pushd _build
+mkdir -p src/%{provider}.%{provider_tld}/%{project}
+ln -s $(dirs +1 -l) src/%{import_path}
+popd
+
+mv vendor src
+
+export GOPATH=$(pwd)/_build:$(pwd)
+export BUILDTAGS='seccomp selinux'
+%gobuild -o bin/%{name} %{import_path}/cmd/%{name}
+#%gobuild -o imgtype %{import_path}/tests/imgtype
+GOMD2MAN=go-md2man %{__make} -C docs
+
+%install
+export GOPATH=$(pwd)/_build:$(pwd):%{gopath}
+make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
+make DESTDIR=%{buildroot} PREFIX=%{_prefix} -C docs install
+
+#install -d -p %{buildroot}/%{_datadir}/%{name}/test/system
+#cp -pav tests/. %{buildroot}/%{_datadir}/%{name}/test/system
+#cp imgtype %{buildroot}/%{_bindir}/%{name}-imgtype
+
+#define license tag if not already defined
+%{!?_licensedir:%global license %doc}
```

Comment:
Not needed (and we always have `%license` defined).
```suggestion
```

Reply:
Removed